src/lib/infrastructure/server/trpc/routers/conversations.ts

```diff
@@ -19,19 +21,42 @@ export const conversationRouter = createTRPCRouter({
         const kpCredentialsDTO = await authGateway.extractKPCredentials();
 
         if (!kpCredentialsDTO.success) {
-            return [];
+            console.error(`Failed to get KP credentials. Dumping DTO: ${kpCredentialsDTO.data.message}`);
```

Comment:
You should use `logger.info` or `logger.debug` for example https://github.com/pinojs/pino